integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java

```diff
@@ -276,12 +277,12 @@ public interface OciConfigBean {
     int imdsTimeoutMilliseconds();
 
     /**
-     * The list of {@link AuthStrategy} names
-     * (excluding {@link AuthStrategy#AUTO}) that
-     * are potentially applicable for use. Here, "potentially applicable or use" means that it is set explicitly by
-     * {@link #authStrategy()}, or else explicitly or implicitly by {@link #authStrategies()}.
+     * The list of {@link AuthStrategy} names (excluding {@link AuthStrategy#AUTO}) that
+     * are potentially applicable for use. Here, "potentially applicable or use" means that it is set using the
```

Comment:
"or use" -> "for use", or just remove the phrase since it adds nothing

Reply:
pushed